tests/transports-integrations/tests/integrations/test_anthropic.py

```
@@ -544,6 +548,21 @@ def test_11_integration_specific_features(self, anthropic_client, test_config):
         # Should prefer calculator for math question
         assert tool_calls[0]["name"] == "calculate"
 
+    @skip_if_no_api_key("anthropic")
+    def test_12_error_handling_invalid_roles(self, anthropic_client, test_config):
```

Ruff (0.11.9)

Missing return type annotation for public function `test_12_error_handling_invalid_roles`

Add return type annotation: `None`

(ANN201)